src/components/Login/Login.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from "react";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import { AUTH_TOKEN } from "../../constants";
+import { Mutation } from "react-apollo";
+import gql from "graphql-tag";
+import { History, LocationState } from "history";
+
+interface Props {
+  history: History<LocationState>;
+}
+interface State {}
+
+const LOGIN_MUTATION = gql`
+  mutation LoginMutation($email: String!, $password: String!) {
+    login(email: $email, password: $password) {
+      token
+    }
+  }
+`;
```

Comment:
I would define all mutations in a `mutations.js` file and import them from there into my components

Reply:
Got it